src/opnsense/mvc/app/views/layouts/default.volt

```diff
@@ -250,7 +250,7 @@
         <!-- page footer -->
     <footer class="page-foot col-sm-push-3 col-lg-push-2">
       <div class="container-fluid">
-        <a target="_blank" href="https://opnsense.org/" class="redlnk">OPNsense</a> (c) 2014-2016 <a href="https://www.deciso.com" class="tblnk">Deciso B.V.</a>
+        <a target="_blank" href="https://opnsense.org/" class="redlnk">OPNsense</a> (c) 2014-2017 <a href="https://www.deciso.com" class="tblnk">Deciso B.V.</a>
```

Comment:
when soft-coding these properties, they probably have to be pushed in somewhere around here (or using setVars in the same file). https://github.com/opnsense/core/blob/17.1.2/src/opnsense/mvc/app/controllers/OPNsense/Base/ControllerBase.php#L219-L222